circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/conf/MetastoreTunnel.java

```diff
@@ -96,7 +96,7 @@ public String getStrictHostKeyChecking() {
   }
 
   public void setStrictHostKeyChecking(String strictHostKeyChecking) {
-    this.strictHostKeyChecking = strictHostKeyChecking == null ? null : strictHostKeyChecking.toLowerCase();
+    this.strictHostKeyChecking = strictHostKeyChecking;
```

Comment:
So you don't need to do the null check and lower casing because this is handled somewhere else?